src/test/java/gov/healthit/chpl/aqa/pageObjects/LoginPage.java

```diff
@@ -4,6 +4,10 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 
+/**
+ * 		11/21/2019	Palak Patel	: Removed and updated welcome Text webelement
```

Comment:
These kinds of changes are better seen using git and git history. Comments like these are only a little helpful when first made, and in 3 to 6 months they'll be so out of date they won't be of any use at all